samples/java/com/microsoft/bing/commerce/samples/ProductSearchSample.java

```diff
@@ -0,0 +1,228 @@
+package com.microsoft.bing.commerce.samples;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import com.microsoft.bing.commerce.search.BingCommerceSearch;
+import com.microsoft.bing.commerce.search.implementation.BingCommerceSearchImpl;
+import com.microsoft.bing.commerce.search.models.*;
+import com.microsoft.bing.commerce.search.util.AccessTokenInterceptor;
+
+public class ProductSearchSample {
+    private final static String TENANT_ID = System.getenv("TENANT_ID");
+    private final static String ACCESS_TOKEN = System.getenv("ACCESS_TOKEN");
+    private final static String indexId = "INDEX_ID";
+    public static void main( String[] args ) throws JsonProcessingException {
+        
+        BingCommerceSearch searchClient = createSearchClient();
+
+        long getMatches = postSearch(searchClient);
+        System.out.format("found [%d] matches with get.\n", getMatches);
+        
+    }
+
+    private static long getSearch(BingCommerceSearch client) {
+        String q="headphones";
+        String tenant=TENANT_ID;
+        String index=indexId; 
+        String mkt=null;
+        String setlang=null;
+        String select="title,brand";
+        String orderby=null;
+        Integer top=20;
+        Integer skip=10;
+        Boolean discoverfacets=false;
+        Boolean alteration=true;
+        Boolean debug = false; 
+        String searchinstanceid=null;
+
+        CommerceSearchResponse response=client.searchs().get(q, tenant, index, mkt, setlang, select, orderby, top, skip, discoverfacets, alteration, debug, searchinstanceid);
+        return response.items().totalEstimatedMatches();
+    }
+
+    private static long postSearch(BingCommerceSearch client) {
+        String matchAll="clothing";
+        List<String> selectItem= Arrays.asList("_itemId", "title");
+
+        RequestQuery query= new RequestQuery().withMatchAll(matchAll);
+
+        CommerceSearchPostRequest request= new CommerceSearchPostRequest().withQuery(query)
+        .withItems(new RequestItems().withSelect(selectItem));
+
+        CommerceSearchResponse response = client.searchs().post(request, TENANT_ID, indexId);
+        return response.items().totalEstimatedMatches();
+    }
+
+    private static long Search_MatchingSpecificFields(BingCommerceSearch client) {
```

Comment:
Java coding style recommends small camel casing  for function names -- all over. ex: `searchMatchingSpecifiedFields`.

Reply:
changed all to camelCase